samples/bluetooth/path_loss_monitoring/README.rst

```diff
@@ -0,0 +1,135 @@
+.. _bluetooth_path_loss_monitoring:
+
+Bluetooth: Path Loss Monitoring
```

Comment:
I'd suggest to have the name without all words starting with capital initial letter:
Path loss monitoring